prepare_release.sh

```diff
@@ -81,3 +83,7 @@ else
 fi
 
 echo "Please merge $RELEASE_BRANCH into $UPSTREAM_BRANCH"
+
+if [[ $DRY_RUN = 1 ]]; then
+  echo "Run these commands to clean up: git co $UPSTREAM_BRANCH && git branch -D $RELEASE_BRANCH && git tag -d $VERSION_PREFIXED"
```

Comment:
Can we use the full commands here => "git checkout" not your alias "git co"? :slightly_smiling_face:

Reply:
Whoops